src/utils.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * This file is part of Serlo.org Cloudflare Worker.
+ *
+ * Copyright (c) 2013-2020 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2013-2020 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link     https://github.com/serlo/serlo.org-cloudflare-worker for the canonical source repository
+ */
+import sanitize from 'sanitize-html'
+import marked from 'marked'
+import { VNode } from 'preact'
+import renderToString from 'preact-render-to-string'
+
+import { NotFound } from './ui'
+
+export const ALL_LANGUAGE_CODES = ['en', 'de', 'fr', 'ta', 'hi', 'es'] as const
+export type LanguageCode = typeof ALL_LANGUAGE_CODES[number]
+
+export function isLanguageCode(code: string): code is LanguageCode {
+  return ALL_LANGUAGE_CODES.some(x => x === code)
+}
+
+export function sanitizeHtml(html: string): string {
```

Comment:
Why? The HTML is in our control

Reply:
Yes, this is a layer of safety in case someone can inject code into the repo `serlo.org-legal`...